web/templates/pages/members.html

```diff
@@ -20,13 +20,13 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
```

Comment:
Every time you make changes to the model, you're modifying the database structure. Hence, [migrations needed](https://docs.djangoproject.com/en/1.7/faq/models/#if-i-make-changes-to-a-model-how-do-i-update-the-database).